lib/dry/cli/command.rb

```diff
@@ -55,6 +55,10 @@ def self.extended(base)
         attr_accessor :subcommands
       end
 
+      # @since 0.6.x
+      # @api private
+      attr_accessor :_context
```

Comment:
@IvanShamatov I would make this `attr_reader` and in the builder method I'd do `instance_variable_set("@_context", context)`. Otherwise it *seems* like this attribute could be changed more than once which I believe is not the case.